app/queues/common/expiration-request.server.ts

```diff
@@ -0,0 +1,101 @@
+import { Worker, Queue, UnrecoverableError } from 'bullmq';
+
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import { prisma } from '~/db.server';
+
+import { addNotification } from '../notifications/notifications.server';
+import { deleteDnsRequest } from '../dns/delete-record-flow.server';
+import { RecordType } from '../notifications/expiration-notification.server';
+
+import type { ExpirationStatusPayload as ExpirationRequestPayload } from '../notifications/expiration-notification.server';
+
+const { EXPIRATION_REPEAT_FREQUENCY_MS, JOB_REMOVAL_FREQUENCY_MS } = process.env;
+const JOB_REMOVAL_INTERVAL = 7 * 24 * 60 * 60 * 1000;
```

Comment:
Add a `_MS` to this as well, and a comment indicating what this magic number equals in hours/days or whatever.